Source/Core/Core/Core.cpp

```diff
@@ -266,11 +272,13 @@ static void ResetRumble()
 // Called from GUI thread
 void Stop()  // - Hammertime!
 {
+  std::string game_id = SConfig::GetInstance().GetGameID();
   if (GetState() == State::Stopping || GetState() == State::Uninitialized)
     return;
 
   s_is_stopping = true;
 
+
```

Comment:
Did you want to write something here? Additional empty lines without anything else feel like you forgot something (or should remove the change from your commit)